lib/utilities/monitoringHandler.js

```diff
@@ -64,6 +64,36 @@ const httpResponseSizeBytes = new client.Summary({
     help: 'Cloudserver HTTP response sizes in bytes',
 });
 
+const quotaEvaluationDuration = new client.Histogram({
+    name: 's3_cloudserver_quota_evaluation_duration_seconds',
+    help: 'Duration of the quota evaluation operation',
+    labelNames: ['action', 'code', 'type'],
+    buckets: [0.0001, 0.005, 0.015, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5],
+});
+
+const utilizationMetricsRetrievalDuration = new client.Histogram({
+    name: 's3_cloudserver_utilization_metrics_retrieval_duration_seconds',
```

Comment:
i think we should keep the same "prefix" for all these metrics, to make it easier to manage the (long) list of metrics...

not sure how best to do it in this case (unless it makes sens to remove the metrics, c.f. earlier comment 😁), the only thing I could come up with is `s3_cloudserver_quota_retrieval_duration_seconds` (which is not _correct_, but probably good enough...)

Reply:
I will use `s3_cloudserver_quota_metrics_retrieval_duration_seconds`

Reply:
See https://github.com/scality/cloudserver/pull/5556/files#diff-3daa33d35f5375a7070d74a0b174c01ab8b982392db4025ebc49cf3690aa16bcR84